charts/opensearch-cluster/templates/opensearch-cluster-cr.yaml

```
@@ -0,0 +1,292 @@
+{{- if eq .Values.OpenSearchClusterSpec.enabled true }}
+apiVersion: opensearch.opster.io/v1
+kind: OpenSearchCluster
+metadata:
+  name: {{ .Values.clusterName | default .Release.Name}}
+  namespace: {{ .Release.Namespace }}
+spec:
+  {{- if .Values.OpenSearchClusterSpec.bootstrap }}
+  bootstrap:
+    {{- if .Values.OpenSearchClusterSpec.bootstrap.additionalConfig }}
+    additionalConfig:
+{{ toYaml .Values.OpenSearchClusterSpec.bootstrap.additionalConfig | indent 6 }}
```

You should indent the line for better readability:
```suggestion
      {{- toYaml .Values.OpenSearchClusterSpec.bootstrap.additionalConfig | indent 6 }}
```
(same for the other ``toYaml` sections)